### PR TITLE
Doc/hfss create port sheet

### PR DESCRIPTION
**Pull request by MaxJPRey**
Add example for the HFSS method create_wave_port_from_sheet.
Add example for the HFSS method create_lumped_port_to_sheet.

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -732,7 +732,7 @@ def create_circuit_port_between_objects(self, startobj, endobject, axisdir="XYNe
             Second (ending) object for the integration line.
         axisdir : str, optional
             Position of the port. It should be one of the values for ``Application.AxisDir``, 
-            which are: ``"XNeg"``, ``"YNeg"``, ``"ZNeg"``, ``"XPos"``, ``"YPos"``, and ``"ZPos"``.
+            which are: ``"XNeg"``, ``"YNeg"``, ``"ZNeg"``, ``"XPos"``, ``"YPos"``, and ``"ZPos"``.
```
Comment (akaszynski) [inline pyaedt/hfss.py:735]:
```suggestion
            which are: ``"XNeg"``, ``"YNeg"``, ``"ZNeg"``, ``"XPos"``, ``"YPos"``, and ``"ZPos"``.
```

File: pyaedt/hfss.py [hunk 2]
```diff
@@ -805,7 +805,7 @@ def create_lumped_port_between_objects(self, startobj, endobject, axisdir=0, imp
         deemb : bool, optional
             Whether to deembed the port. The default is ``False``.
         port_on_plane : bool, optional
-            Whether to create the source on the plane orthogonal to ``AxisDir``.
+            Whether to create the source on the plane orthogonal to ``AxisDir``.
```
Comment (akaszynski) [inline pyaedt/hfss.py:808]:
```suggestion
            Whether to create the source on the plane orthogonal to ``AxisDir``.
```

File: pyaedt/hfss.py [hunk 3]
```diff
@@ -864,7 +864,7 @@ def create_voltage_source_from_objects(self, startobj, endobject, axisdir=0, sou
             Second (ending) object for the integration line.
         axisdir : str, optional
             Position of the port. It should be one of the values for ``Application.AxisDir``, 
-            which are: ``"XNeg"``, ``"YNeg"``, ``"ZNeg"``, ``"XPos"``, ``"YPos"``, and ``"ZPos"``.
+            which are: ``"XNeg"``, ``"YNeg"``, ``"ZNeg"``, ``"XPos"``, ``"YPos"``, and ``"ZPos"``.
```
Comment (akaszynski) [inline pyaedt/hfss.py:867]:
```suggestion
            which are: ``"XNeg"``, ``"YNeg"``, ``"ZNeg"``, ``"XPos"``, ``"YPos"``, and ``"ZPos"``.
```

File: pyaedt/hfss.py [hunk 4]
```diff
@@ -1013,7 +1013,7 @@ def create_wave_port_between_objects(self, startobj, endobject, axisdir=0, imped
...
```
Comment (akaszynski) [inline pyaedt/hfss.py:1016]:
```suggestion
            which are: ``"XNeg"``, ``"YNeg"``, ``"ZNeg"``, ``"XPos"``, ``"YPos"``, and ``"ZPos"``.
```

File: pyaedt/hfss.py [hunk 5]
```diff
@@ -1109,7 +1109,7 @@ def create_wave_port_microstrip_between_objects(self, startobj, endobject, axisd
         Parameters
         ----------
         startobj : 
-            First (starting) object for the integration line. This is typically the reference plane.
+            First (starting) object for the integration line. This is typically the reference plane.
```
Comment (akaszynski) [inline pyaedt/hfss.py:1112]:
```suggestion
            First (starting) object for the integration line. This is typically the reference plane.
```

File: pyaedt/hfss.py [hunk 6]
```diff
@@ -1625,8 +1625,24 @@ def create_wave_port_from_sheet(self, sheet, deemb=0, axisdir=0, impedance=50, n
...
+        Examples
+        --------
+
+        Create a circle sheet that will be used to create a wave port named
+        ``'WavePortFromSheet'``.
+
+        >>> origin_position = hfss.modeler.Position(0, 0, 0)
+        >>> circle = hfss.modeler.primitives.create_circle(hfss.CoordinateSystemPlane.YZPlane,
+        ...                                                origin_position, 10, name="WaveCircle")
+        >>> hfss.solution_type ="DrivenModal"
```
Comment (akaszynski) [inline pyaedt/hfss.py:1638]:
```suggestion
        >>> hfss.solution_type = "DrivenModal"
```

File: pyaedt/hfss.py [hunk 7]
```diff
@@ -2108,6 +2138,7 @@ def edit_source(self, portandmode, powerin, phase="0deg"):
         >>> sheet = hfss.modeler.primitives.create_circle(hfss.CoordinateSystemPlane.YZPlane,
         ...                                               [-20, 0, 0], 10,
         ...                                               name="sheet_for_source")
+        >>> hfss.solution_type ="DrivenModal"
```
Comment (akaszynski) [inline pyaedt/hfss.py:2141]:
```suggestion
        >>> hfss.solution_type = "DrivenModal"
```

**Review by akaszynski — APPROVED** [review]
Minor whitespace nitpicks, otherwise LGTM.

**Comment by MaxJPRey** [comment]
Thanks @akaszynski for the review.

**Pull request merged**